docs/compose/SectionHeader/README.md

```diff
@@ -0,0 +1,36 @@
+# SectionHeader
+
+[![Maven Central](https://img.shields.io/maven-central/v/net.skyscanner.backpack/backpack-compose)](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose)
+[![Class reference](https://img.shields.io/badge/Class%20reference-Android-blue)](https://backpack.github.io/android/backpack-compose/net.skyscanner.backpack.compose.sectionheader)
+[![Source code](https://img.shields.io/badge/Source%20code-GitHub-lightgrey)](https://github.com/Skyscanner/backpack-android/tree/main/backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/sectionheader)
+
+## Default
```

Comment:
If you keep having multiple stories please include all screenshots here (and generate the screenshots please)

Reply:
done ✅